pkg/os/launchd/launchd_darwin.go

```diff
@@ -103,7 +104,11 @@ func StopAgent(label string) error {
 
 // RestartAgent restarts a launchd agent
 func RestartAgent(label string) error {
-	err := StopAgent(label)
+	err := LoadPlist(label)
```

Comment:
Not clear why we need this change, if the agent was already running, then wasn't the plist also loaded already?

Reply:
This was a left over from debug/testing, fixed now!